diff --color -ur a/source/dialogs/drun.c b/source/dialogs/drun.c
--- a/source/dialogs/drun.c	2021-11-15 18:57:53.390558933 +0100
+++ b/source/dialogs/drun.c	2021-11-15 19:00:14.314940730 +0100
@@ -1017,6 +1017,18 @@
       g_free(dir);
       TICK_N("Get Desktop apps (user dir)");
     }
+	{
+      gchar *dir;
+      // Second read nix-profile dir
+      dir = g_build_filename ( g_get_home_dir (), ".nix-profile/share/applications", NULL );
+      walk_dir ( pd, dir, dir );
+      g_free ( dir );
+      
+      // Third read nix-profile dir
+      dir = g_build_filename ( "/run/current-system/sw/share/applications", NULL );
+      walk_dir ( pd, dir, dir );
+      g_free ( dir );
+	}
 
     /** Load application entires */
     p = rofi_theme_find_property(wid, P_BOOLEAN, "parse-system", TRUE);
